clients/unshaded/src/main/java/tachyon/client/file/FileInStream.java

```diff
@@ -145,7 +151,7 @@ public int read(byte[] b, int off, int len) throws IOException {
         try {
           mCurrentCacheStream.write(b, currentOffset, bytesRead);
         } catch (IOException ioe) {
-          // TODO(yupeng): Log debug maybe?
+          LOG.error("Failed to write into buffer: " + ioe.getMessage());
```

Comment:
Not sure if this warrants an `error`, maybe `warn` since the operation doesn't fail in this case.